apps/web/components/eventtype/CreateEventType.tsx

```diff
@@ -56,12 +56,22 @@ export default function CreateEventTypeButton(props: Props) {
       : undefined;
   const pageSlug = router.query.eventPage || props.options[0].slug;
   const hasTeams = !!props.options.find((option) => option.teamId);
+  const title: string =
+    typeof router.query.title === "string" && router.query.title ? router.query.title : "";
+  const length: number =
+    typeof router.query.length === "string" && router.query.length ? parseInt(router.query.length) : 15;
+  const description: string =
+    typeof router.query.description === "string" && router.query.description ? router.query.description : "";
```

Comment:
Instead of using a ternary statement here we can do something like 
`router.query.description ?? "" ` Might be a little cleaner

Reply:
The issue with this is that the query can be a string array as well. Hence the string comparison